src/main/java/com/tngtech/configbuilder/util/ConfigValidator.java

```diff
@@ -7,9 +7,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.validation.ConstraintViolation;
-import javax.validation.Validator;
-import javax.validation.ValidatorFactory;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
```

Comment:
I wouldn't call the commit `chore(deps)`, but rather `BREAKING CHANGE`, as it changes the `ConfigValidator` to use Jakarta Bean Validation 3.0 / Jakarta EE 9, right?

Should we release `config-builder` version 2.0 after this update?
Should there be a version 1.7.1 released before this is merged?

Reply:
Good question. On the one hand, this is just a dependency update, the  config-builder API does not change at all, unless you want to acknowledge people using custom validation logic in their project, but that is not really something that worries me.
I thought about upgrading to Hibernate 8 directly, but that would indeed require a newer JRE version, a true breaking change. 
Maybe merge the rest first, release an 1.7.x version, then update Hibernate and release 1.8. 2.0 would be a JDK11-compatible version.

Reply:
I think that it _is_ breaking, because I assumed that the new `jakarta.validation.Validator` wouldn't validate `javax.validation.constraints` (which users might currently use), cf. `ConfigBuilderIntegrationTest`.

IMO, we transitively expose the Bean Validation API.

Reply:
Yes, the user would need to keep two different Hibernate versions, and both jakarta.validation and javax.validation around, and exclude the transitive dependency from config-builder if necessary. That is probably enough to constitute a breaking change.
Will update the version accordingly and let this PR sit for a while.